### PR TITLE
Feat: Add bundle size calculation on comments

### LINKED ISSUE #139: Add bundle size calculation comments
/bounty $15

### PR DESCRIPTION
**Pull request by divanshu-go**
Closes #139 
/claim #139

### CODE REVIEW

File: .github/workflows/bundle-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+name: Bundle Size Analysis
+
+on:
+  pull_request:
+    branches: [main]
+    paths:
+      - 'lib/**'
```
Comment (imrishabh18) [inline .github/workflows/bundle-analysis.yml:7]: I think we need to trigger this, only when new dependencies are added or removed. So, I think only the `package.json` and `bun.lockb` should be there in paths.

cc: @seveibar

**Review by imrishabh18 — CHANGES_REQUESTED** [review]

**Comment by divanshu-go** [comment]
@imrishabh18 I tested this workflow here on my fork
image

**Comment by divanshu-go** [comment]
@imrishabh18 @seveibar  this is the link to my PR on my forked repository.
https://github.com/divanshu-go/circuit-to-svg/pull/1#issuecomment-2509025663

**Comment by divanshu-go** [comment]
@seveibar can u please review here also.

**Comment by seveibar** [comment]
@imrishabh18 is correct. Also it needs to work here, not on your fork. Other repositories use a proxy url to make it work on repos that aren't just a fork. If it only works on forks it's not useful- there is a proxy url that other repos use. Take a look at e.g. snippets

**Comment by divanshu-go** [comment]
@seveibar can u please trigger the workflows here? so that we can check.

**Comment by seveibar** [comment]
@divanshu-go hmm i didn't see that. This repo now has actions enabled for contributors, thanks for point out it didn't run

**Comment by divanshu-go** [comment]
@seveibar is this correct now .

**Comment by divanshu-go** [comment]
@seveibar no worries.

**Comment by divanshu-go** [comment]
@seveibar so can we merge it now.

**Comment by seveibar** [comment]
@divanshu-go we should still make sure that this only runs on package.json and bun.lockb changes or whatever the other repos check against as @imrishabh18 mentioned. Going to leave this one to @imrishabh18 b/c he has context 🫡

File: .github/workflows/bundle-analysis.yml (hunk 1 above)
Comment (divanshu-go) [reply .github/workflows/bundle-analysis.yml]: done @imrishabh18

**Review by imrishabh18 — APPROVED** [review]
LGTM!

**Pull request merged**